src/components/Table/TableHeader/TableHeader.module.scss

```diff
@@ -1,12 +1,12 @@
 .tableHeader {
   display: grid;
   grid-template-columns: var(--table-grid-template-columns);
+  position: sticky;
+  top: 0;
+  z-index: 1;
+  background-color: inherit;
```

Comment:
isn't inherit the default behavior?

Reply:
nope, defaults to transparent (makes sense, it is the broader use case, since otherwise we might have to configure transparent background for each subelement where we define background for parent)